src/binance/bnbEngine.ts

```diff
@@ -52,9 +52,10 @@ export class BinanceEngine extends CurrencyEngine<BinanceTools> {
     tools: BinanceTools,
     walletInfo: EdgeWalletInfo,
     initOptions: any, // BinanceInitOptions,
-    opts: any // EdgeCurrencyEngineOptions
+    opts: EdgeCurrencyEngineOptions,
+    env: PluginEnvironment<{}>
```

Comment:
Optional: In the `makeCurrencyEngine` function that the the innerPlugin system calls, the parameter order is:

```
(
  env: PluginEnvironment<...>,
  tools: EosTools,
  walletInfo: EdgeWalletInfo,
  opts: EdgeCurrencyEngineOptions
```

It would be nice if the engine constructors used the same order, followed by whatever coin-specific stuff they want. So `env` would go first in all these edits, instead of last.

Reply:
Ack. Pretty please can we make this a separate PR. There are just so many of these assets to modify and I have just one more day to get this in.